src/components/ScholarshipForm.jsx

```diff
@@ -24,51 +21,24 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-function ScholarshipForm({ setSubmissionAlert }) {
+function ScholarshipForm({ scholarship, submitFn, onSubmitError }) {
   const classes = useStyles();
   const user = firebase.auth().currentUser;
 
   const formik = useFormik({
-    initialValues: {
-      name: '',
-      deadline: null,
-      description: '',
-      amount: {
-        type: null,
-        min: 0,
-        max: 0,
-      },
-      website: '',
-    },
+    initialValues: scholarship.data,
     validationSchema,
     onSubmit: (values, { setSubmitting, resetForm }) => {
       setSubmitting(true);
-      Scholarships.new({
+      scholarship.data = {
         ...values,
-        author: {
-          id: user?.uid,
-          email: user?.email,
-        },
-      })
+        author: { id: user?.uid, email: user?.email },
```

Comment:
This is something that no longer belongs here as this form could be used for updating existing scholarships and this would change the author.

Reply:
I moved it to `AddScholarship`